conductor/src/module.cpp

```diff
@@ -0,0 +1,18 @@
+
+#include "isolate.hpp"
+
+using namespace cocaine;
+using namespace cocaine::isolate;
+
+extern "C" {
+    auto
+    validation() -> api::preconditions_t {
+        return api::preconditions_t { COCAINE_MAKE_VERSION(0, 12, 0) };
+    }
+
+    void
+    initialize(api::repository_t& repository) {
+        repository.insert<conductor::isolate_t>("conductor");
```

Comment:
Well, naming is up to discussion. Conductor is a thing that connects (isolated) isolates, or, otherwise, a thing that makes them conduct, which both makes sense to me somehow :)